index.html

```diff
@@ -4201,6 +4207,458 @@ <h3>Device Theft and Impersonation</h3>
     </section>
   </section>
 
+  <section class="appendix">
+    <h2>Proof Format Trade-Offs</h2>
+
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging syntaxes and digital proof formats. Each
+format has benefits and drawbacks. The following table is intended to summarize
+a number of these native trade-offs.
+    </p>
+
+    <p>
+The table below compares three proof format ecosystems; JSON+JWTs,
+JSON-LD+JWTs, and JSON-LD+LD-Proofs. At the time of publication of this
+specification, no organization is planning to use JSON+JWTs. The existence
+of JSON+JWTs in the table below is provided based on review comments on the
+specification for readers that were curious about the benefits that JSON-LD
+provided over JSON. Readers should also be aware that Zero-Knowledge Proofs
+are currently proposed as a sub-type of LD-Proofs and thus fall into the
+final colum below.
+    </p>
+
+    <table class="simple">
+      <thead>
+        <tr>
+          <th style="text-align: center;">Feature</th>
+          <th style="text-align: center;">JSON<br>+<br>JWTs</th>
+          <th style="text-align: center;">JSON&#8209;LD<br>+<br>JWTs</th>
+          <th style="text-align: center;">JSON&#8209;LD<br>+<br>LD&#8209;Proofs</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+<a href="#pf1">PF1.</a> Support for open world data modelling.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf2">PF2.</a> Universal identifier mechanism for JSON objects via
```

Comment:
Interesting in a comparison between JSON and JSON-LD. But not in a comparison between proof formats.

Reply:
@pelle wrote: 
> Our base layer at uPort is JSON+JWT

Given your comment above, I was under the mistaken impression that no implementer had chosen to make JSON+JWT their base layer. Now that you've corrected that misunderstanding, I have renamed the section "Syntax and Proof Format Trade-Offs" for two reasons 1) reviewers wanted to know why they would choose JSON vs. JSON-LD, and 2) reviewers want to know why they would choose JWTs vs LD-Proofs.